### ISSUE TITLE
Move accessibility checker to plugin

### ISSUE THREAD

**Issue by bjcfuller** [uriweb/uri-modern#170]
## Issue Type

[] Bug  
[] New Feature  
[X] Enhancement  
    
## Description

Accessibility and code style checking ("shaming") should be moved out of the theme and into a standalone plugin.

**Comment by bjcfuller** [comment]
Fixed in a6cd0c37643998ff9b56202830ea0fb7fd40b7e5
